openshift/system/Dockerfile

```diff
@@ -1,24 +1,37 @@
-FROM quay.io/centos/centos:stream8 as builder
-WORKDIR /opt/system
+FROM quay.io/centos/centos:stream8 as base
+
 ENV RUBY_MAJOR_VERSION=2 \
     RUBY_MINOR_VERSION=7 \
     RAILS_ENV=production \
-    NODE_ENV=production
+    NODE_ENV=production \
+    SAFETY_ASSURED=1 \
+    TZ=:/etc/localtime \
+    HOME=/opt/system
+
 ENV RUBY_VERSION="${RUBY_MAJOR_VERSION}.${RUBY_MINOR_VERSION}"
 
+WORKDIR $HOME
+
 ADD . ./
 RUN cp config/examples/*.yml config/ \
     && cp openshift/system/config/* config/
 
 RUN sed -i 's/mirrorlist/#mirrorlist/g' /etc/yum.repos.d/CentOS-* \
     && sed -i 's|#baseurl=http://mirror.centos.org|baseurl=http://vault.centos.org|g' /etc/yum.repos.d/CentOS-* \
-    && dnf -y --setopt=module_stream_switch=True module enable ruby:${RUBY_VERSION} nodejs:16 mysql:8.0 \
-    && dnf install -y --setopt=skip_missing_names_on_install=False,tsflags=nodocs shared-mime-info make automake gcc gcc-c++ redhat-rpm-config postgresql rubygem-irb rubygem-rdoc ruby-devel nodejs libpq-devel mysql-devel gd-devel libxml2-devel libxslt-devel git 'dnf-command(download)' podman-catatonit \
```

Comment:
All `devel` packages are installed in the `builder` image, all the others are installed here. We should not install the same packages twice.